charts/icm-web/.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.6.0
+current_version = 1.0.0-alpha
```

Comment:
Why do you switch to version 1.x ?

Reply:
The values.yaml changed. I thinks this is a api-breaking thing. Nevertheless I now changed to 0.9.0-alpha